ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <react/renderer/components/view/ViewProps.h>
+
+namespace facebook {
+namespace react {
+
+// TODO (T28334063): Consider for codegen.
```

Comment:
Is this TODO Valid here ? Though it's been copied from frame files, as we are not using codgen, looks like not valid here.

Reply:
Addressed